memory/table_data.go

```diff
@@ -276,6 +276,14 @@ func (td *TableData) numRows(ctx *sql.Context) (uint64, error) {
 // throws an error if any two or more rows share the same |cols| values.
 func (td *TableData) errIfDuplicateEntryExist(cols []string, idxName string) error {
 	columnMapping, err := td.columnIndexes(cols)
+
+	// We currently skip validating duplicates on unique virtual columns.
+	for _, i := range columnMapping {
```

Comment:
Should have a skipped test assertion for this

Reply:
Done, and added comments.

Unfortunately the assertion will get skipped for both GMS and Dolt, but I don't think we have a way to just disable it for GMS.